install/kubernetes/helm/istio/charts/kiali/templates/gateway.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-kiali-gateway
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    istio: {{ .Values.gatewayName }}
```

Comment:
It's not required that the selector be `istio: <name>`, it could be arbitrary labels. We should change `values.yaml` to take selector, and provide default value `istio: ingressgateway`.

Reply:
@gyliu513 sure I can hardcode the selector and agree it would be better to do so.  Less chance for an operator to crater their install.  I'm not sure why @douglas-reid chose to make this configurable.  Will remove that configurability across the install codebase in a followup PR.

Reply:
@ZackButcher sure - makes sense.  I do agree with @gyliu513 's position that we should simplify this part by hardcoding.  I don't see any good reason to make the selector here given the nature of exposing services via ingressgateway.  This saves on the expense of introducing another LoadBalancer service type, and the logistics of managing that IP.